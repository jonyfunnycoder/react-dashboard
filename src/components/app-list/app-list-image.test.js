import React from 'react';
import renderer from "react-test-renderer";
import Avtar from "../../images/avtar.png";
import AppListImage from "./app-list-image";

test('renders app list image correctly ', () => {
    const tree = renderer
        .create(<AppListImage url={Avtar} />)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
