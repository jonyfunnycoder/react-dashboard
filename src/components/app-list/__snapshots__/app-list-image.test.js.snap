// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app list image correctly  1`] = `
<img
  alt="test"
  className="avtar"
  src="avtar.png"
/>
`;
